platform/mellanox/docker-syncd-mlnx/base_image_files/monit_syncd

```diff
@@ -3,5 +3,5 @@
 ## process list:
 ##  syncd
 ###############################################################################
-check process syncd matching "/usr/bin/syncd\s"
-    if does not exist for 5 times within 5 cycles then alert
+check program syncd|syncd with path "/usr/bin/process_checker syncd syncd /usr/bin/syncd --diag"
```

Comment:
Is this tested? It seems like this reintroduces the issue that we fixed with https://github.com/Azure/sonic-buildimage/pull/4706. Not all vendors pass the same flags to syncd